### PR TITLE
Add best practice documentation for interop

### PR DESCRIPTION
**Pull request by garethbowen**
Expands the existing interop page with more information and best practice about how it should be done today.

### CODE REVIEW

File: content/en/_index.md [hunk 1]
```diff
@@ -47,7 +47,6 @@ The CHT provides you with resources to design, build, deploy, and monitor digita
 
 ## Getting Started
 
-<br />
```
Comment (garethbowen) [inline content/en/_index.md:50]: This is unrelated but fixes a rendering issue on the page.

File: content/en/apps/concepts/interoperability.md [hunk 2]
```diff
@@ -4,13 +4,35 @@ linkTitle: "Interoperability"
...
+- [OpenHIE](https://ohie.org/) defines the architecture for an interoperability layer.
+- [OpenHIM](http://openhim.org/) is a middleware component designed to ease interoperability between systems.
+- [HL7 FHIR](https://www.hl7.org/fhir/index.html) is a messaging format to allow all systems to understand the format of the message.
+- [ICD-11](https://www.who.int/standards/classifications/classification-of-diseases) and [LOINC](https://loinc.org/) are examples of classification systems.
+
+# CHT
+
+The structure of documents in the CHT database reflect the configuration of the system, and therefore do not map directly to a FHIR message format. To achieve interoperability you should use middleware to convert the CHT datastructure into a standardized form so the other systems can read it.
+
+{{< figure src="../interoperability.png" link="../interoperability.png" >}}
```
Comment (mrjones-plip) [inline content/en/apps/concepts/interoperability.md:31]: not sure if it's documented, but existing docs image protocol is to create a directory (eg `interoperability`) and then put the image in there. You'll have to update the `src` to match.  See here [for example](https://github.com/medic/cht-docs/blame/interoperability-best-practice/content/en/apps/concepts/workflows.md#L22).

File: content/en/apps/concepts/interoperability.md [hunk 3]
```diff
@@ -4,13 +4,35 @@ linkTitle: "Interoperability"
...
   apps/features/integrations
 ---
 
-*Interoperability* and *Integration* are often used interchangeably but mean different things. [This article](https://www.linkedin.com/pulse/lets-talk-interoperability-vs-integration-mike-fleck) offers a useful delineation between the two.
+# Introduction
 
-With **interoperability**, systems work together, even if they weren’t designed specifically to work together, because standard communication protocol techniques are applied. Two (or more) systems that weren't designed to be interoperable require **integration**.
+Interoperability is the ability of health information systems to work together, even if they weren't specifically designed to work together. With interoperability, patient information can be seen, exchanged, and used across different platforms. This is different from _integration_ which requires custom development to connect two specific systems together. 
+
+Interoperability is the best practice for health systems because it allows information from one system to be shared with one or more other systems for with no additional development.
```
Comment (mrjones-plip) [inline content/en/apps/concepts/interoperability.md:16]: I couldn't find one in a few min of searching - but a canonical "why interop is needed for CHIS success" link here would be good.  Not required!

**Review by mrjones-plip — APPROVED** [review]
Great work here !! I actually felt relieved to be reading it ; )

I made two minor suggestions, but these don't block merging, and you can fix at your discretion. 

lgtm!

File: content/en/apps/concepts/interoperability.md (hunk 2 above)
Comment (garethbowen) [reply content/en/apps/concepts/interoperability.md:31]: Done.

File: content/en/apps/concepts/interoperability.md (hunk 3 above)
Comment (garethbowen) [reply content/en/apps/concepts/interoperability.md:16]: It's a little scary but I couldn't find one either. I'm going to leave it for now and keep iterating on this when I'm back...

**Pull request merged**